pkg/bpf/perf.go

```diff
@@ -138,6 +138,35 @@ import (
 const (
 	EventsMapName   = "cilium_events"
 	MAX_POLL_EVENTS = 32
+
+	PERF_TYPE_HARDWARE   = 0
```

Comment:
This is a strict copy of the header `<linux/perf.h>` and the values must be in sync at all times. I don't want to use iota because it creates the impression that the value is defined on the go side which it is not. These values will never ever change again.